server/utils/licenceStatus.js

```diff
@@ -194,13 +202,31 @@ function getExclusionTaskState(licence) {
 }
 
 function getCrdTimeState(licence) {
-
-    const timeAnswer = getIn(licence, ['eligibility', 'crdTime', 'decision']);
+    const decision = getIn(licence, ['eligibility', 'crdTime', 'decision']);
+    const dmApproval = getIn(licence, ['eligibility', 'crdTime', 'dmApproval']);
 
     return {
-        insufficientTime: timeAnswer === 'Yes',
-        crdTime: timeAnswer ? taskStates.DONE : taskStates.UNSTARTED
+        insufficientTimeContinue: decision === 'Yes' && dmApproval === 'Yes',
+        insufficientTimeStop: decision === 'Yes' && dmApproval === 'No',
+        insufficientTime: decision === 'Yes',
+        crdTime: getState(licence)
     };
+
```

Comment:
This isn't clear, and the way insufficientTimeContinue is used in eligibilityTask isn't consistent.
Would suggest something like
```
    return {
        insufficientTime: decision === 'Yes',
        insufficientTimeApproval: dmApproval === 'Yes',

        insufficientTimeStop: decision === 'Yes' && dmApproval === 'No',
        
        crdTime: getState(licence)
    };
```

insuffientTime and insufficientTimeApproval are used to convey the decisions for the two inividual questions based on their answers, whereas insufficientTimeStop is the overall effect of the two things combined.

In the eligibilityTask, this
```
if licenceStatus.decisions.insufficientTimeContinue
                                            | Yes
                                        else
                                            | No
```
won't show the chosen answer, but instead shows the combination